driver-core/src/test/unit/com/mongodb/internal/connection/LoggingCommandEventSenderSpecification.groovy

```diff
@@ -41,6 +41,7 @@ import spock.lang.Specification
 
 import static com.mongodb.connection.ClusterConnectionMode.MULTIPLE
 import static com.mongodb.connection.ClusterConnectionMode.SINGLE
+import static com.mongodb.internal.operation.ServerVersionHelper.LATEST_WIRE_VERSION
 import static com.mongodb.internal.operation.ServerVersionHelper.THREE_DOT_SIX_WIRE_VERSION
```

Comment:
Is `ServerVersionHelper.THREE_DOT_SIX_WIRE_VERSION` needed anymore?

Reply:
It was just used in a few tests, so I replaced all usages with LATEST_WIRE_VERSION and removed it.